tests/TestHarness/Cluster.py

```diff
@@ -1731,6 +1731,6 @@ def waitForTrxGeneratorsSpinup(self, nodeId: int, numGenerators: int, timeout: i
                     firstTrxs.append(line.rstrip('\n'))
         Utils.Print(f"first transactions: {firstTrxs}")
         status = node.waitForTransactionsInBlock(firstTrxs)
-        if status is None:
+        if status is None or status is False:
```

Comment:
Wondering if this would be better?
```suggestion
        if not status:
```